### ISSUE TITLE
Changelog documentation Diff Link error

### ISSUE THREAD

**Issue by brentm5** [chef-cookbooks/audit#66]
Not sure how else to surface this issue and its not that big of a deal but your changelog generator is producing broken links:

https://github.com/chef-cookbooks/audit/blame/master/CHANGELOG.md#L4

```
# This line
[Full Changelog](https://github.com/chef-cookbooks/audit/compare/v0.12.0...0.13.0)

# Should be this
[Full Changelog](https://github.com/chef-cookbooks/audit/compare/v0.12.0...v0.13.0)
                                                            Notice the V ->^
```

Not sure how to fix it myself or I would

**Comment by chris-rock** [comment]
Thanks for highlighting this issue @bigbam505

**Comment by chris-rock** [comment]
fixed by #74
